back-end/src/test/java/com/skynet/skymed/model/PessoaTest.java

```diff
@@ -0,0 +1,214 @@
+package com.skynet.skymed.model;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class PessoaTest {
+
+	private Pessoa pessoa;
+
+	@Before
+	public void setUp() throws Exception {
+
+		pessoa = inicializarPessoa();
+
+	}
+
+	private Pessoa inicializarPessoa() throws Exception {
+
+		var pessoa = new Pessoa();
+		var endereco = new Endereco();
+
+		endereco.setCep("74922-740");
+		endereco.setComplemento("Qd 127 Lt. 9");
+		endereco.setIBGE("12345");
+		endereco.setLocalidade("Aparecida de Goiânia");
+		endereco.setLogradouro("Rua X 29C");
+		endereco.setNumero("23");
+		endereco.setUf("GO");
+
+		pessoa.setNome("Raphael Mota");
+		pessoa.setEmail("raphael.mota14@hotmail.com");
+		pessoa.setEhPaciente(true);
+		pessoa.setSenha("1234");
+		pessoa.setTelefone("(62) 9272-5598");
+		pessoa.setRg("5946186");
+		pessoa.setEndereco(endereco);
+		pessoa.setCpf_cnpj("707.772.711-41");
+		pessoa.setTokenDeAutenticacao("1234");
+		pessoa.setTokenDeRedefinicao("1244");
+
+		return pessoa;
+	}
+
+	@Test
+	public void InsereCamposInvalidos() {
+
+		try {
+			pessoa.setCpf_cnpj(null);
```

Comment:
```suggestion
			pessoa.setCpf_cnpj(null);
			
			Assert.fail();
```
Faz isso pra TODOS os casos por favor(se vc não fizer, pode ser q não de uma Exception ai passa como caso correto, essa linha nunca vai executar se o código acima lançar uma Exception)